src/components/info/elements/UserAvatar.js

```diff
@@ -0,0 +1,29 @@
+import ItemLoader from "@components/attach/elements/ItemLoader";
+import { useMemo } from "react";
+
+export default function UserAvatar({
+  size = 1,
+  avatarUrl,
+  avatarBlurHash,
+  defaultIcon,
+}) {
+  const defaultSize = 80;
+
+  const avatarView = useMemo(
+    () =>
+      avatarUrl ? (
+        <img src={avatarUrl} alt={"UserAvatra"} />
```

Comment:
A good alt value should be descriptive and helpful, especially for users who rely on screen readers
 
 Let's do this:

`alt={"UserAvatra"}` -> `alt={"User's Profile Picture"}`